mlos_bench/mlos_bench/optimizers/base_optimizer.py

```diff
@@ -40,7 +41,8 @@ def __init__(self, tunables: TunableGroups, service: Optional[Service], config:
         self._tunables = tunables
         self._service = service
         self._iter = 1
-        self._max_iter = int(self._config.pop('max_iterations', 10))
+        self._use_defaults: bool = bool(strtobool(str.lower(self._config.pop('use_defaults', "True"))))
+        self._max_iter = int(self._config.pop('max_iterations', 100))
```

Comment:
10 iterations is difficult to get much optimization out of, so I bumped up the default here, but we can back that out and leave it for another time if we want.

Reply:
yeah, I would probably keep some low number here, so that the user doesn't have to wonder why the program takes so long to finish. After all, I expect that `max_iterations` will be set explicitly in pretty much every config

Reply:
Agreed.  I set it to 25 for now.  10 is barely into the random iterations.